src/model/emojie.ts

```diff
@@ -0,0 +1,15 @@
+import {Entity, PrimaryGeneratedColumn, Column, ManyToOne} from 'typeorm';
+import { User } from './User'
+import { Algorithem } from './Algorithem'
+
+@Entity()
+export class Emojie {
```

Comment:
```suggestion
export class Emoji {
```
오타가 있는 것 같네요. 전체적으로 `Emoji`로 수정해주세용

Reply:
확인했습니다